doc/source/whatsnew/v0.24.0.txt

```diff
@@ -630,7 +630,7 @@ Numeric
 Strings
 ^^^^^^^
 
--
+- Removed as_indexer(deprecated of 0.21.0) keyword completely from str.match() (:issue:`22356`,:issue:`6581`)
```

Comment:
* You mean `0.20.0` ?  You can also just remove that parenthetical comment.
* This should be in the `Removal of prior version deprecations/changes` section.

Reply:
thanks, will fix it straight away